src/routes.js

```diff
@@ -34,29 +34,11 @@ import ControlAddContainer from './containers/control/add/Connect'
 import App from './containers/app'
 import ObservationControlGridShow from './containers/observationcontrol/grid'
 
-export default (store) => {
```

Comment:
I too think that this is not needed anymore. But please give it a second thought. Is it because of containers/app/index.js logic for calling loadUser and redirecting if not loaded, that you removed this?